.github/workflows/update-tooling.yml

```diff
@@ -2,6 +2,7 @@ name: Update Tooling
 on:
   schedule:
     - cron: "0 2 * * *"
+  workflow_dispatch:
```

Comment:
```suggestion
  workflow_dispatch
```

Reply:
I believe a colon is required.  Otherwise throws a cannot read implicit mapping error

Reply:
It should probably be `workflow_dispatch: {}` then, but if this works it's fine.